cmd/collector/main.go

```diff
@@ -25,6 +27,9 @@ func main() {
 	cmpts, err := defaults.Components()
 	handleErr(err)
 
+	es := elasticsearch.Factory{}
```

Comment:
Might be good to group the Jaeger components, as done by the core repo and contrib repo, so other projects could easily include them as a group?